docs/DeveloperGuide.adoc

```diff
@@ -720,22 +720,42 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
-
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|Priority |As a ... |I want to ... |So that I...
+|`* * *` |new user |see usage instructions. |refer to instructions when I forget how to use the App.
+|`* * *` |user |add a new person. |
+|`* * *` |user |delete a person. |remove entries that I no longer need.
+|`* * *` |user |find a person by name. |locate details of persons without having to go through the entire list
+|`* * *` |user |click a link to access files sent to me by a contact. |do not have to search my email/dropbox for it.
```

Comment:
Take note that one of the project constraints is to have the primary input being CLI.